src/api/public/apidocs-new/paths/published_project_name_repository_name.yaml

```diff
@@ -0,0 +1,35 @@
+get:
+  summary: List the architectures of the pair project/repository with published binaries.
+  description: >
+    Get a list of the architectures of the given pair project and repository that already have published binaries.
```

Comment:
@saraycp I think that not a 100% correct, to me it looks more like the directory tree of the download repository/place where the binaries get published. Here an example https://download.opensuse.org/repositories/OBS:/Server:/Unstable/SLE_12_SP5/

Reply:
Its basically what you described here, but one level up https://github.com/openSUSE/open-build-service/pull/10975/files#diff-941da4259ba92323e54fa18add5045aeed9001f3e26c9e7b4040e9c335f25331R103

Reply:
You are completely right. I'm going to give a better description. One more time we are in a case where the name is not completely appropriate. 

The output of `/published/{project_name}/{repository_name}` can be something like:

```
<directory>
  <entry name="repocache"/>
  <entry name="repodata"/>
  <entry name="src"/>
  <entry name="x86_64"/>
</directory>
```
Then we can "inspect" one of them with the following endpoint:

`/published/{project_name}/{repository_name}/{architecture_name}` where `{architecture_name}` can be replaced by  `src` or `repocache` that are not "architectures".